recipes/open62541/all/submoduledata.yml

```diff
@@ -0,0 +1,6 @@
+submodules:
```

Comment:
I have a mechanism for downloading submodules in this recipe, would it make sense to add an option to tools.get or add a new function in conan for it ?

Reply:
I think it would make sense when you use the scm feature of conan, which is forbidden on cci.
The aim of cci is to provide packages which can be consumed using conan alone.

So in this case, mdnsd should be *conanized* and added as a conan dependency.

Reply:
In the general case it would make sense, but it points to specific version of mdnsd, no idea if it matches a stable version (according to mdnsd itself) at all.

Reply:
I didn't notice before, but it is even worse : it point to a fork of the original mdnsd.

Reply:
When in doubt: ask, or look what the linux distributions are doing
It looks like the fork has added cmake support.
Maybe the goal of the fork is only to provide cmake support.

Reply:
Differences to the base repo (https://github.com/troglobit/mdnsd):

    * Use CMake for the build
    * Strict compilation flags for better portability
    * Support of Linux, MinGW, OS X and Windows
    * Continuous Integration

Reply:
The base repo is way more active.
And looking at recent commits, actual bugs are fixed.

It is possible to build using autotools with MSVC when using appropriate wrapper scripts.
Maybe the changes of the *Pro* fork can help what changes should be made.
If possible, they can be upstreamed so the autotools build system improves.

Reply:
OK understood, but I find it better to do in two steps here. Have the opcua package and improve msdns after because I feel like there is much work.

Reply:
Actually it passes the hooks checks now.

[OPCUA](https://en.wikipedia.org/wiki/OPC_Unified_Architecture) is an industrial protocol. It is related to the IoT world. This library is the most popular open source implementation.